Clarify difference between compositions and associations (#715)
cap-js/docs: advanced/fiori.md

:::

### Difference between Compositions and Associations

> Be aware that all compositions of a draft-enabled entity are part of the same draft. Only those entities will have a "Create" button in SAP Fiori elements UIs, as they are part of the draft. Conversely, associated entities must not be modified through drafts. It is only possible to modify foreign keys if they are stored within the draft entity.

Reviewer: ```suggestion
Be aware that you should not modify associated entities through drafts. Only compositions which are part of the same draft will get a "Create" button in SAP Fiori elements UIs.
```
I think it's more that they shouldn't because it's applied immediately it breaks the "draft" feature but that's my guess. I tried to rephrase it, @David-Kunz Does this convey the whole meaning? If the last sentence is saying something in addition, I can add it back. Like this perhaps:
```

> Be aware that all compositions of a draft-enabled entity are part of the same draft. Only those entities will have a "Create" button in SAP Fiori elements UIs, as they are part of the draft. Conversely, associated entities must not be modified through drafts. It is only possible to modify foreign keys if they are stored within the draft entity.

Reviewer 2: @mariayord Could you please have a look and comment? Thanks :)

> Be aware that all compositions of a draft-enabled entity are part of the same draft. Only those entities will have a "Create" button in SAP Fiori elements UIs, as they are part of the draft. Conversely, associated entities must not be modified through drafts. It is only possible to modify foreign keys if they are stored within the draft entity.

Reviewer 2: @David-Kunz Could you have a look at the sugestion?

> Be aware that all compositions of a draft-enabled entity are part of the same draft. Only those entities will have a "Create" button in SAP Fiori elements UIs, as they are part of the draft. Conversely, associated entities must not be modified through drafts. It is only possible to modify foreign keys if they are stored within the draft entity.

Author: Hi @renejeglinsky , I think `must not` is better, since they must not create such scenarios in the first place.

> Be aware that all compositions of a draft-enabled entity are part of the same draft. Only those entities will have a "Create" button in SAP Fiori elements UIs, as they are part of the draft. Conversely, associated entities must not be modified through drafts. It is only possible to modify foreign keys if they are stored within the draft entity.

Author: and modified == updated, deleted, created

> Be aware that all compositions of a draft-enabled entity are part of the same draft. Only those entities will have a "Create" button in SAP Fiori elements UIs, as they are part of the draft. Conversely, associated entities must not be modified through drafts. It is only possible to modify foreign keys if they are stored within the draft entity.

Reviewer 2: So how Maha rephrased, just using most not would be good?
